charts/prometheus-thanos/README.md

```diff
@@ -233,9 +233,8 @@ The following table lists the configurable parameters of the prometheus-thanos c
 | `storeGateway.image.repository` | Docker image repo for store gateway | `quay.io/thanos/thanos` |
 | `storeGateway.image.tag` | Docker image tag for store gateway | `v0.10.1` |
 | `storeGateway.image.pullPolicy` | Docker image pull policy for store gateway | `IfNotPresent` |
-| `storeGateway.serviceAccount.create` | Create service account | `true` |
-| `storeGateway.serviceAccount.annotations` | Service account annotations | `nil` |
-| `storeGateway.indexCacheSize` | Index cache size | `500MB` |
+| `storeGateway.indexCache.config` | Config for the index cache, see [the docs](https://thanos.io/components/store.md/#index-cache) | `500MB` for the maximum size |
```

Comment:
"`500MB` for the maximum size |" seems wrong there.

Reply:
I think this replicates the behaviour of the previous verison of this chart which set `index-cache-size=500MB` (https://thanos.io/components/store.md/#in-memory-index-cache). Happy to change it to a more sensible value. Do you have anything in mind?
Or would you like me to be clearer on the description?

Reply:
The description is good but teh default values "500MB for the maximum size" seems wrong.

Reply:
Updated again. Thanks for having a look. 👍